model_bakery/baker.py

```diff
@@ -619,3 +614,36 @@ def filter_rel_attrs(field_name: str, **rel_attrs) -> Dict[str, Any]:
             clean_dict[k] = v
 
     return clean_dict
+
+
+def bulk_create(baker, quantity, **kwargs) -> None:
+    """
+    Bulk create entries and all related FKs as well.
+
+    Important: there's no way to avoid save calls since Django does
+    not return the created objects after a bulk_insert call.
+    """
+
+    def _save_related_objs(model, objects) -> None:
+        fk_fields = [
+            f
+            for f in model._meta.fields
+            if isinstance(f, OneToOneField) or isinstance(f, ForeignKey)
+        ]
+
+        for fk in fk_fields:
+            fk_objects = []
+            for obj in objects:
+                fk_obj = getattr(obj, fk.name, None)
+                if fk_obj and not fk_obj.pk:
+                    fk_objects.append(fk_obj)
+
+            if fk_objects:
+                _save_related_objs(fk.related_model, fk_objects)
+                for i, fk_obj in enumerate(fk_objects):
+                    fk_obj.save()
+                    setattr(objects[i], fk.name, fk_obj)
```

Comment:
What if the `fk_objects` do not match up 1-to-1 with `objects`? (One way the code looks like this could happen is if the condition on line 638 is not `True` and a given `object`'s related model for this iteration's `fk` is not appended to the list. Could that ever happen? If not, then is the condition on line 638 necessary?

Reply:
I added the condition on 638 just as an explicit safeguard. For example, that condition enables people to write tests such as:

```
user = baker.make(User)
logs = baker.make(LogEntries, user=user, _quantity=10, _bulk_create=True)
```

Every log entry will already have a associated user with and `id` because it has already been previously created. If we remove 638 condition, baker can perform unexpected triggers on the `user` object. For example, what if's there a `post_save` method on the User model.

I mean, I just think it was safer to add this check to not perform extra and unnecessary save operations.